src/invidious/routes/preferences.cr

```diff
@@ -222,10 +222,10 @@ module Invidious::Routes::PreferencesRoute
 
       if CONFIG.domain
         env.response.cookies["PREFS"] = HTTP::Cookie.new(name: "PREFS", domain: "#{CONFIG.domain}", value: URI.encode_www_form(preferences.to_json), expires: Time.utc + 2.years,
-          secure: secure, http_only: true)
+          secure: secure, http_only: false)
```

Comment:
Could you please rebase against master, as I merged a PR that changed that code?

Reply:
Rebased